terminal.go

```diff
@@ -62,6 +67,10 @@ func (t *Terminal) SetHistoryFile(f string) error {
 		log.Infof("No history file specified")
 		return nil
 	}
+	if t.capacity <= 0 {
+		log.Infof("No history capacity set, ignoring history file %s", f)
+		return nil
+	}
 	if !t.IsTerminal() {
 		log.Infof("Not a terminal, not setting history file")
```

Comment:
Warning, or debug as I'm unsure this information is needed

Reply:
make up your mind :) warning is higher level than info
it's always useful to confirm what's happening in term of flow, and one can run `-quiet` if desired

also ... this is a sample